projs/shadow/shadow-engine/core/inc/debug/DebugModule.h

```diff
@@ -18,6 +13,12 @@ namespace ShadowEngine::Debug {
         bool active;
 
     public:
+        DebugModule() : Module("module:/debug/debugger") {}
+
+        std::vector<std::string> GetDependencies() override {
```

Comment:
I can't help but feel like this should be set by the module zip, rather than the code.